apps/cookbook/src/app/component-status/component-status.component.scss

```diff
@@ -126,7 +126,8 @@
     }
   }
 
-  a[href^="https://"] ion-icon {
+  a[href^="https://"] ion-icon
+  {
```

Comment:
Do you have any idea why prettier did this?

Reply:
Not 100% sure about the root cause, but it has to do with the double forward slash. 
If I escape them, the style rule still works as expected, and prettier treats it like the rest of the code.

![image](https://user-images.githubusercontent.com/42470636/147944096-34c3f314-e74f-4fce-860b-cf17cb192a4e.png)

Reply:
I'm using this solution for now, let me know if you think it should be reverted.